ptr/value.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
```

Comment:
the linter also does not like that this does not match what is in https://github.com/knative/pkg/blob/main/hack/boilerplate/boilerplate.go.txt